backend/gzdb/db.go

```diff
@@ -485,7 +485,31 @@ func (g *gormZenaoDB) GetEventUsersWithRole(eventID string, role string) ([]*zen
 func (g *gormZenaoDB) GetEventUserTickets(eventID string, userID string) ([]*zeni.SoldTicket, error) {
 	userIDint, err := strconv.ParseUint(userID, 10, 64)
 	if err != nil {
-		return nil, fmt.Errorf("parse buyer id: %w", err)
+		return nil, fmt.Errorf("parse user id: %w", err)
+	}
+
+	tickets := []*SoldTicket{}
+	err = g.db.Model(&SoldTicket{}).Preload("Checkin").Preload("User").Find(&tickets, "event_id = ? AND user_id = ?", eventID, userIDint).Error
```

Comment:
we could miss tickets with no user_id (which is possible for anon tickets), we should fetch all tickets of event during gentxs instead